automation/trestle.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+TRESTLE_IMAGE="localhost:5000/trestle-demo"
+TRESTLE_IMAGE_TAG="latest"
+
+TRESTLE_CMD_IMAGE="trestle"
+PANDOC_CMD_IMAGE="/demo/bin/pandoc"
+PYTHON_CMD_IMAGE="python"
+
+CONTAINER_CMD=$(command -pv podman || command -pv docker)
+TRESTLE_IMAGE_EXISTS=$($CONTAINER_CMD image exists $TRESTLE_IMAGE; echo $?)
```

Comment:
`docker` doesn't have `image exists` so suggesting inspect instead.

```suggestion
TRESTLE_IMAGE_EXISTS=$($CONTAINER_CMD image inspect $TRESTLE_IMAGE 2> /dev/null; echo $?)
```